custom_components/tuya_local/devices/myuet_me82_thermostat.yaml

```diff
@@ -0,0 +1,215 @@
+name: Thermostat
+products:
+  - id: 7ingxzrs0wo9rqsd
+    name: Myuet ME82
+primary_entity:
+  entity: climate
+  dps:
+    - id: 1
+      type: boolean
+      name: hvac_mode
+      mapping:
```

Comment:
Is there any reason this does not combine with the mode, the same way the me81 config does?

Reply:
Yes, for some reason both my thermostats don't respond if I perform more than one action at a time. Changing the mode sets the temperature and the mode and nothing happens, doing both independently works fine